drgn/helpers/linux/stackdepot.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) Google LLC
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+"""
+Stack Depot
+-----------
+
+The ``drgn.helpers.linux.stackdepot`` module provides helpers for working with
+the stack depot implementation used by KASAN and other kernel debugging tools.
+"""
+
+from typing import Optional
+
+from drgn import Object, StackTrace, cast, reinterpret
+
+__all__ = ("stack_depot_fetch",)
+
+
+def stack_depot_fetch(handle: Object) -> Optional[StackTrace]:
+    """
+    Returns a stack trace for the given stack handle.
+
+    :param handle: ``depot_stack_handle_t``
+    :return: The stack trace, or ``None`` if not available.
+    """
+    prog = handle.prog_
+    handle_parts = reinterpret("union handle_parts", handle)
+
+    # Renamed in commit 961c949b012f ("lib/stackdepot: rename slab to pool").
```

Comment:
[This should include](https://github.com/osandov/drgn/blob/main/CONTRIBUTING.rst#linux-kernel-helpers) the kernel version, too.
```suggestion
    # Renamed in Linux kernel commit 961c949b012f ("lib/stackdepot: rename slab
    # to pool") (in v6.3).
```